cmd/frontend/internal/app/ui/handlers.go

```diff
@@ -256,6 +275,36 @@ func serveSignIn(w http.ResponseWriter, r *http.Request) error {
 	return renderTemplate(w, "app.html", common)
 }
 
+// redirectTreeOrBlob redirects a blob page to a tree page if the file is actually a directory,
```

Comment:
```suggestion
// Redirects a blob page to a tree page if the file is actually a directory,
```

just reduced redundancy in the comments, discard if you don't like it

Reply:
I prefer the no redundancy part, but I think this is the convention in Go 😄 
I think it's to distinguish line comments from symbol docblocks. Rename symbol will also update the comments in Go

Reply:
>since Go has no multi-line comments

It does :)

```go
/*
	A
	multi
	line
	comment
*/
```

But yes having the symbol name as the first word is the convention.